fe/fe-core/src/main/java/org/apache/doris/analysis/SlotDescriptor.java

```diff
@@ -338,6 +338,7 @@ public String getExplainString(String prefix) {
         builder.append(prefix).append("byteOffset=").append(byteOffset).append("\n");
         builder.append(prefix).append("nullIndicatorByte=").append(nullIndicatorByte).append("\n");
         builder.append(prefix).append("nullIndicatorBit=").append(nullIndicatorBit).append("\n");
+        builder.append(prefix).append("nullable=").append(isNullable).append("\n");
```

Comment:
i don't think this is a good idea to add this in explain string since BE do not use this attribute anymore. It could mislead users if it show different nullable info with nullIndicatorByte and nullIndicatorBit

Reply:
I think it's easier to understand though, in many scenes, the only thing which be cared about is just if the slot is nullable...